cerebrium/faqs-and-help/fast-deployments-dos-and-donts.mdx

```diff
@@ -10,13 +10,13 @@ Whether you're a seasoned developer or a tech enthusiast venturing, these guidel
 
 In this brief guide, we delve into the strategies, best practices, and pitfalls to avoid when it comes to the deployment process.
 
-### Setting up your requirements.txt
+### Setting up your cerebrium.requirements
 
 - **Do** freeze the versions of your Python libraries.  
-  Solving the requirements for your project can extend your build time unnecessarily. During the development process, your hard work setting up an environment for your model has already determined which versions you need for each of your requirements. Use these versions when creating your requirements.txt
-- **Don't** include built-in modules in your requirements file as this wastes time during your build.
+  Solving the requirements for your project can extend your build time unnecessarily. During the development process, your hard work setting up an environment for your model has already determined which versions you need for each of your requirements. Use these versions when creating your cerebrium.requirements in your cerebrium.toml file.
+- **Don't** include built-in modules in your requirements as this wastes time during your build.
```

Comment:
cerebrium.requirements?

Reply:
I think my this more obvious, eg "Setting up Dependencies in your Project" or something along those lines